more_itertools/more.py

```diff
@@ -175,6 +197,55 @@ def peek(self, default=_marker):
                 return default
         return self._cache[0]
 
+    def prepend(self, *items):
+        """Stack up items to be the next ones returned from ``next()`` or
+        ``self.peek()``. The items will be returned in first-in first-out order:
+
+            >>> p = peekable([1, 2, 3])
+            >>> p.prepend(10, 11, 12)
+            >>> next(p)
+            10
+            >>> list(p)
+            [11, 12, 1, 2, 3]
+
+        If you want to prepend elements such that the last one in the argument
```

Comment:
I'd still rather not spell it out - note how the [the deque docs](https://docs.python.org/3/library/collections.html#collections.deque.extendleft) handle this. Since the ordering might be surprising they mention it, but leave the handling of it to the reader.

If you still think it's important, could you pick one method - either `reverse` or `prepend`-ing single items in a loop? If we're going to explain it, then I'd rather do it one way than two (PEP 20 styles).